### PR TITLE
Added TypeScript types.

### PR DESCRIPTION
**Pull request by nickytonline**
This adds TypeScript types. We'd need these for completing the transition to TypeScript for Refined GitHub, https://github.com/sindresorhus/refined-github/pull/1783

### CODE REVIEW

**Comment by nickytonline** [comment]
The build is failing, but has nothing to do with the types. I can fix those issues in another PR if you want.

**Comment by sindresorhus** [comment]
Thanks for creating a TypeScript definition. 🙌 Can you follow [this styleguide](https://github.com/sindresorhus/typescript-definition-style-guide)?

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare const copyToClipboard: { (data: string): boolean; };
+
+export default copyToClipboard;
```
Comment (sindresorhus) [inline index.d.ts]: Would be simpler to just use `export default function copyToClipboard ...`

Comment (nickytonline) [reply index.d.ts]: Yeah, that's what I had initially. Will revert.

File: index.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ module.exports = input => {
 		originalRange = selection.getRangeAt(0);
 	}
 
-	document.body.appendChild(el);
+	document.body.append(el);
```
Comment (nickytonline) [inline index.js:21]: Fixed issue that failing the build. Not related to types addition.

File: index.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ module.exports = input => {
 	let success = false;
 	try {
 		success = document.execCommand('copy');
-	} catch (err) {}
+	} catch (error) {}
```
Comment (nickytonline) [inline index.js:31]: Fixed issue that failing the build. Not related to types addition.

**Comment by nickytonline** [comment]
No rush @sindresorhus, but just wondering if there is anything else you'd like me to do for this PR.

**Comment by sindresorhus** [comment]
I went with #24 as it was more complete

**Pull request closed without merging**